ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The opentelemetry-ext-dbapi package allows tracing queries made by the
+ibraries following Ptyhon Database API specification:
+https://www.python.org/dev/peps/pep-0249/
+"""
+
+import typing
+
+import wrapt
+
+from opentelemetry.trace import SpanKind, Tracer
+from opentelemetry.trace.status import Status, StatusCanonicalCode
+
+QUERY_WRAP_METHODS = ["execute", "executemany", "callproc"]
```

Comment:
It seems like the OpenCensus integration does not patch `callproc`, any reason why this is included? Also it seems like not all databases provide stored procedures (https://www.python.org/dev/peps/pep-0249/#id42)

Reply:
No clue why OpenCensus doesn't have it, in OpenTelemetry we do care about stored procedures  
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-database.md#semantic-conventions-for-database-client-calls

Added extra check to see if the method exist to avoid crashing when is not there, thanks for the link